### DISCUSSION TITLE
After clicking an application in the menu, there is no visual feedback, nor the menu disappears.

### DISCUSSION THREAD

**Discussion by anthonyreds** [Hofknecht/SystemTrayMenu#525] [Q&A]
Is expected that the menu doesn't give any visual feedback or reaction until the selected app shows up in front?

**Comment by topeterk** [comment]
In the released version (v1) the icon should show the loading circle for a short moment:
![v1](https://github.com/Hofknecht/SystemTrayMenu/assets/4932525/9674f872-7423-426b-8df2-fd9928f630ad)
https://github.com/Hofknecht/SystemTrayMenu/assets/4932525/d90b8a61-f331-4856-8c37-075c4d4d62bf

In v2 we might do something else. As of now it shows a short moving pulse in the background:
![v2](https://github.com/Hofknecht/SystemTrayMenu/assets/4932525/8e76ebc3-b59e-4d5c-8d8e-079b7ca60d01)
https://github.com/Hofknecht/SystemTrayMenu/assets/4932525/06498449-6c5c-4a50-81ea-5b5e447d868a
Not sure, if we will backport any animation to v1.

But when I understood it correctly, this should already be working this way.
